.gitignore

```diff
@@ -0,0 +1,4 @@
+bin/
+obj/
+*.pdb
+*.csproj.user
```

Comment:
*.suo could also go here.

Reply:
@danwell Could you add the *.suo as mentioned and pull from the master please? I have added a license file.

Reply:
@vvidetta, @Alexei-B: Done. Removed '*.pdb' as well as that should be covered by obj/, bin/.